### PR TITLE
[MIG] partner_zone module migration to 11.0

### PR DESCRIPTION
**Pull request by hprasetyou**

### CODE REVIEW

File: partner_zone/__init__.py
Comment (Daniel-CA) [inline partner_zone/__init__.py]: [ the # -*- coding: utf-8 -*- line is not necessary anymore in Python 3](https://stackoverflow.com/questions/37048761/coding-utf-8-on-python3/37048998#37048998)

**Comment by Daniel-CA** [comment]
:+1: LGTM, thanks for your work! If you can clean a bit the commit history it will be perfect.

**Comment by oihane** [comment]
Please squash commits into one you can follow this tutorial
https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by hprasetyou** [comment]
sorry, a little bit confused :grinning: how about now?

**Review by oihane — APPROVED** [review]

**Review by Daniel-CA — APPROVED** [review]

**Pull request merged**
